base/lib/migrate.mustache

```diff
@@ -11,7 +11,7 @@ const r = require('lib/db');
 const table = r.table('_migrations');
 
 const applyMigration = (migrations, then) => {
-  if (migrations.length === 0) return;
+  if (migrations.length === 0) return Promise.resolve();
```

Comment:
There is a consistent return rule. If in a function, nothing is returned, all functions must return nothing. If something is returned all functions must return something.